ttnn/cpp/pybind11/operations/core.hpp

```diff
@@ -19,51 +19,57 @@ namespace core {
 void py_module(py::module& module) {
     module.def(
         "reshape",
-        [](const ttnn::Tensor& tensor, const ttnn::Shape& shape) -> ttnn::Tensor {
-            return ttnn::reshape(tensor, shape);
+        [](const ttnn::Tensor& tensor, const ttnn::Shape& shape, const std::optional<MemoryConfig>& memory_config) -> ttnn::Tensor {
+            return ttnn::reshape(tensor, shape, memory_config);
```

Comment:
why do we need memory_config conversions in reshape? What's the use-case?

Reply:
It has been raised as an issue here
https://github.com/tenstorrent/tt-metal/issues/6350
So I have added it

Reply:
@nemanjagrujic @sjameelTT can you please resolve this?

Reply:
Hello, the ticket reports and we noticed that some ttnn operations miss memory_config parameter while almost all of the operations have it (And lists the operations where the parameter is missing). 

If that behaviour is expected (and not by mistake), that's fine and can be responded in ticket comments.